modules/packages/LocalAtomics.chpl

```diff
@@ -0,0 +1,354 @@
+/*
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module LocalAtomics {
+  /*
+     Planned usage:
+     var head : LocalAtomicObject(unmanaged Node(int));
+  // Adding a new node...
+  head.write(new unmanaged Node(int)); // Need 'write(objType)'
+  */
+
+  extern {
```

Comment:
@dgarvit Can you add this before the `export`...

```chpl
if CHPL_TARGET_ARCH != "x86_64" {
   compilerWarning("Program compiled for ", CHPL_TARGET_ARCH, " but target should be x86_64");
}
```

Reply:
Done.